openmc/cmfd.py

```diff
@@ -0,0 +1,3072 @@
+            dx = self._hxyz[:,:,:,0][self._accel_neig_left_idxs]
+            vals = (-1.0 * dtilde - dhat) / dx
+            # Store data to add to CSR matrix
+            data = np.append(data, vals)
+
+            # Define leakage terms that relate terms to their neighbors to the
+            # right
+            dtilde = self._dtilde[:,:,:,g,1][self._accel_neig_right_idxs]
+            dhat = self._dhat[:,:,:,g,1][self._accel_neig_right_idxs]
+            dx = self._hxyz[:,:,:,0][self._accel_neig_right_idxs]
+            vals = (-1.0 * dtilde + dhat) / dx
+            # Store data to add to CSR matrix
+            data = np.append(data, vals)
+
+            # Define leakage terms that relate terms to their neighbors in the
+            # back
+            dtilde = self._dtilde[:,:,:,g,2][self._accel_neig_back_idxs]
+            dhat = self._dhat[:,:,:,g,2][self._accel_neig_back_idxs]
+            dy = self._hxyz[:,:,:,1][self._accel_neig_back_idxs]
+            vals = (-1.0 * dtilde - dhat) / dy
+            # Store data to add to CSR matrix
+            data = np.append(data, vals)
+
+            # Define leakage terms that relate terms to their neighbors in the
+            # front
+            dtilde = self._dtilde[:,:,:,g,3][self._accel_neig_front_idxs]
+            dhat = self._dhat[:,:,:,g,3][self._accel_neig_front_idxs]
+            dy = self._hxyz[:,:,:,1][self._accel_neig_front_idxs]
+            vals = (-1.0 * dtilde + dhat) / dy
+            # Store data to add to CSR matrix
+            data = np.append(data, vals)
+
+            # Define leakage terms that relate terms to their neighbors to the
+            # bottom
+            dtilde = self._dtilde[:,:,:,g,4][self._accel_neig_bot_idxs]
+            dhat = self._dhat[:,:,:,g,4][self._accel_neig_bot_idxs]
+            dz = self._hxyz[:,:,:,2][self._accel_neig_bot_idxs]
+            vals = (-1.0 * dtilde - dhat) / dz
+            # Store data to add to CSR matrix
+            data = np.append(data, vals)
+
+            # Define leakage terms that relate terms to their neighbors to the
+            # top
+            dtilde = self._dtilde[:,:,:,g,5][self._accel_neig_top_idxs]
+            dhat = self._dhat[:,:,:,g,5][self._accel_neig_top_idxs]
+            dz = self._hxyz[:,:,:,2][self._accel_neig_top_idxs]
+            vals = (-1.0 * dtilde + dhat) / dz
+            # Store data to add to CSR matrix
+            data = np.append(data, vals)
+
+            # Define terms that relate to loss of neutrons in a cell. These
+            # correspond to all the diagonal entries of the loss matrix
+            jnet_g = jnet[:,:,:,g][self._accel_idxs]
+            total_xs = self._totalxs[:,:,:,g][self._accel_idxs]
+            scatt_xs = self._scattxs[:,:,:,g,g][self._accel_idxs]
+            vals = jnet_g + total_xs - scatt_xs
+            # Store data to add to CSR matrix
+            data = np.append(data, vals)
+
+            # Define terms that relate to in
```

Comment:
Question: Just to confirm, `12` is related to the number (or indices) of only accelerated regions?